target/linux/kirkwood/image/Makefile

```diff
@@ -34,6 +34,7 @@ endef
 
 define Device/dockstar
   DEVICE_DTS := kirkwood-dockstar
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage
   FILESYSTEMS := squashfs
```

Comment:
Actually, this is unnecessary. USB2 support (including mass-storage support) is built into the kernel